### PR TITLE
Remove presentation classes from grid code samples

### LINKED ISSUE #906: Styleguide presentation classes in code samples
In some cases, classes only intended for presentation of the styleguide are being shown in the code sample markup. This is bc our code samples are live and automatically generated by the actual markup. This should be removed so people can directly copy and paste the code samples without getting these extraneous classes.

One place where this occurs is in the grids. You'll see classes `usa-grid-example` and `usa-grid-example`.

screen shot 2015-12-03 at 10 55 46 am

One reason why this happens is bc we have two differently styled examples in a single preview box. So we can't target these wthout adding a class inside our around the markup.

Ideally, this is what users would see in the code sample:

```html
  <div class="usa-grid">
    <div class="usa-width-one-whole">1/1</div>
  </div>
  <div class="usa-grid">
    <div class="usa-width-one-half">1/2</div>
    <div class="usa-width-one-half">1/2</div>
  </div>
  <div class="usa-grid">
    <div class="usa-width-one-third">1/3</div>
    <div class="usa-width-one-third">1/3</div>
    <div class="usa-width-one-third">1/3</div>
  </div>
```

### PR DESCRIPTION
**Pull request by maya**
This is the same PR as #913. This adds a new code sample box below the first grid example in order to remove extraneous classes.

Resolves #906.

### CODE REVIEW

File: assets/css/styleguide.scss [hunk 1]
```diff
@@ -442,24 +444,28 @@ $site-top:           124px;
...
-  background: $color-grid-dark;
+.grid-text {
+  .usa-grid {
+    background: $color-grid-dark;
 
-  > * {
-  background: $color-grid-light;
-  color: #000;
-  }
+    > * {
```
Comment (msecret) [inline assets/css/styleguide.scss]: This is hard to see in gh, but shouldn't the lines below be indented in one more?

Comment (maya) [inline assets/css/styleguide.scss]: @msecret I just checked.. it's indented with the background property which is one level in from `.usa-grid` so I think that's correct:

screen shot 2015-12-18 at 10 55 48 am

Comment (maya) [inline assets/css/styleguide.scss]: @msecret Ooops, looked at the wrong line. Fixing....

Comment (maya) [inline assets/css/styleguide.scss]: Fixed in 5ccaffe.

**Pull request merged**